product/dialogs/miq_dialogs/miq_provision_vmware_dialogs_template.yaml

```diff
@@ -421,6 +421,14 @@
           :required: false
           :display: :edit
           :data_type: :integer
+        :storage_profile_filter:
```

Comment:
This key name should probably be `:storage_profile`.  As mentioned before the other fields designated with `_filter` are global or user-defined filters for the different resources.  The `StorageProfile` is an actual resource like data-center, host or resource_pool.